### DISCUSSION TITLE
Dim desktop, except the active window

### DISCUSSION THREAD

**Discussion by gmmazza** [WindowTop/WindowTop-App#274] [Ideas]
To help maintain focus on big screens, have the option to enable dim the brightness of the desktop, apart from the active window. 
Inspiration in this https://le-dimmer.en.lo4d.com/screenshots. 
Moreover, for further development of the idea, have a specific window that enable the dimming and opacity or colour separated by window based.

**Comment by gileli121** [comment]
Thank you for the idea.
It makes sense to include it as part of WindowTop.

**Comment by maxually** [comment]
Just came here to suggest this exact thing. Similarly, being able to set the transparency for the border separately from the transparency of the window, so that you can put the window to 100% transparent and have a nice bold border around it.

**Reply by gileli121** [reply]
This was actually considered a bug (https://github.com/WindowTop/WindowTop-App/issues/299) that was fixed in the last version.
I could not imagine that someone would think about it as a feature.

If you want to make the border more transparent, you can do it from here:
![image](https://github.com/WindowTop/WindowTop-App/assets/17680514/6e3cb1df-b0da-4f2b-a77e-9badcd5bc3b7)

The A slider allows you to make it more transparent.
But after the bugfix, the border can't be more transparent than the window.
